doc/getting-started.rst

```diff
@@ -70,25 +70,22 @@ As mentioned above, ``cabal init`` with no arguments generates a package with a
 single executable that prints ``"Hello, Haskell!"`` to the terminal. To run the
 executable enter the following command:
 
-``cabal run :myfirstapp``
+``cabal run myfirstapp``
 
 You should see the following output in the terminal:
 
 .. code-block:: console
 
-     $ cabal run :myfirstapp
+     $ cabal run myfirstapp
      ...
      Hello, Haskell!
 
-.. note:: The ``:`` prefix in ``:myfirstapp`` signifies that the
-	  ``myfirstapp`` target is part of the current package.
-
```

Comment:
I wonder what was meant by this. @m-renaud did it work in the past or did you intend to write `.:myfirstapp`?

Reply:
This is actually accepted syntax 💀 

```shell
λ P haskell → mkdir tlkajdfd
λ P haskell → cd tlkajdfd 
λ P tlkajdfd → cabal init

Guessing dependencies...

Generating LICENSE...
Warning: unknown license type, you must put a copy in LICENSE yourself.
Generating CHANGELOG.md...
Generating app/Main.hs...
Generating tlkajdfd.cabal...

Warning: no synopsis given. You should edit the .cabal file and add one.
You may want to edit the .cabal file and add a Description field.
λ P tlkajdfd → ls
CHANGELOG.md   app            tlkajdfd.cabal
λ P tlkajdfd → cabal run :tlkajdfd
Resolving dependencies...
Build profile: -w ghc-8.10.5 -O1
In order, the following will be built (use -v for more details):
 - tlkajdfd-0.1.0.0 (exe:tlkajdfd) (first run)
Configuring executable 'tlkajdfd' for tlkajdfd-0.1.0.0..
Preprocessing executable 'tlkajdfd' for tlkajdfd-0.1.0.0..
Building executable 'tlkajdfd' for tlkajdfd-0.1.0.0..
[1 of 1] Compiling Main             ( app/Main.hs, /Users/emilypi/haskell/tlkajdfd/dist-newstyle/build/x86_64-osx/ghc-8.10.5/tlkajdfd-0.1.0.0/x/tlkajdfd/build/tlkajdfd/tlkajdfd-tmp/Main.o )
Linking /Users/emilypi/haskell/tlkajdfd/dist-newstyle/build/x86_64-osx/ghc-8.10.5/tlkajdfd-0.1.0.0/x/tlkajdfd/build/tlkajdfd/tlkajdfd ...
Hello, Haskell!
λ P tlkajdfd → 
```

Yeah no - let's not introduce people to that.

Reply:
@emilypi I'm getting the same error as in #7460 when running your commands with cabal-install and Cabal-the-library both version 3.4.0.0.

Reply:
Weird. I'm running the same: 

```
λ P tlkajdfd → cabal --version
cabal-install version 3.4.0.0
compiled using version 3.4.0.0 of the Cabal library 
λ P tlkajdfd → ghc --version
The Glorious Glasgow Haskell Compilation System, version 8.10.5
λ P tlkajdfd → 
```

What distro?

Reply:
I'm running NixOS 21.05 and GHC 8.10.4.

Reply:
Ahh. I'm on Macos, and it works for me. I can get to my ubuntu box in a few days, but I wonder if @m-renaud wrote this while on a Mac!